packages/common-all/src/sidebar.ts

```diff
@@ -0,0 +1,389 @@
+import _ from "lodash";
```

Comment:
does this code belong in `common-all` or in `common-frontend`?

Reply:
[`e1b192c` (#3448)](https://github.com/dendronhq/dendron/pull/3448/commits/e1b192c0a64aba6cd6e7d1050c4441b4e406a2b1)

through this I recognized I forgot checking for `nav_exclude_children` and `has_collection`. I added that check here: [`9071f62` (#3448)](https://github.com/dendronhq/dendron/pull/3448/commits/9071f629e8b50df4255b16b471df8311090a1e1e)

It would be also possible to move that logic into `packages/common-all/src/sidebar.ts`. I think it would be good that only sidebar.ts is responsible for managing that kind of domain.

Reply:
these comments should have gone under https://github.com/dendronhq/dendron/pull/3448#discussion_r966169516

regarding:

> does this code belong in common-all or in common-frontend?

I am not sure. For me it fits more in common-all because it has more to do with backdround processes and less with frontend, at least not in the direct sense. `sidebar.ts` is used in the export nextjs pod directly.